CHANGELOG.adoc

```diff
@@ -57,6 +57,7 @@
 - [diagram] Fix an issue preventing the resizing of a node if the cursor had not moved after a previous resizing
 - https://github.com/eclipse-sirius/sirius-components/issues/1075[#1075] [compatibility] Use the proper icon feature for OperationAction
 - https://github.com/eclipse-sirius/sirius-components/issues/1104[#1104] [diagram] Fix the image base path during the SVG export
+- https://github.com/eclipse-sirius/sirius-components/issues/1104[#1102] [diagram] Fix the node label positioning (rectangle, list and list item)
```

Comment:
The URL is not valid, which is why the build is failing. Please keep an eye on the results of the build of your pull requests